modules/odc-interacting-with-applications-and-components.adoc

```diff
@@ -5,21 +5,21 @@
 [id="odc-interacting-with-applications-and-components_{context}"]
 = Interacting with the application and the components
 
-The *Topology* view in the *Developer* perspective of the web console provides the following options to interact with the application and the components:
+By default, the *Topology view* renders in the *Developer* perspective of the web console provides the following options to interact with the application and the components:
 
-* Click *Open URL* (image:odc_open_url.png[title="Application Link"]) to see your application exposed by the route on a public URL.
-* Click *Edit Source code* to access your source code and modify it.
+* Use the *List View* icon to see a list of all your applications and use the *Topology View* icon to switch back to the *Topology* view.
+** Click *Open URL* (image:odc_open_url.png[title="Application Link"]) to see your application exposed by the route on a public URL.
```

Comment:
On top of this bullet point you can add: The *Topology view* provides the following options to interact with the application and the components:
and sorry, but can we move the bullet points back to level 1.